index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../styles/style-main.css">
+    <title>Setup and Mobile Version Skeleton</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <nav class="navbar">
+        <ul>
+            <li><a href="#">Hello</a></li>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+    </nav>
+
+    <main>
+
+        <section class="headline">
+
+            <div class="headline-left">
+
+                <h1 class="headline-name">
+                    Charlie <br>
+                    Giorgania
+                </h1>
+
+                <p class="headline-bio">Hello! I am a software developer! <br>
+                    I can help you build a product, feature or website. <br>
+                    Taka a look of my works. <br>
+                    If you like what you see and have a project you need coded, <br>
+                    don’t hesitate and contact me.</p>
+
+                <button id="start-collaboration" type="button">Start collaboration</button>
+                <img id="dots" src="/images/image_dots.png" alt="Square made out of dots">
+
+            </div>
+
+            <div class="headline-right">
+                <img id="shapes" src="/images/image-geometry_1.png" alt="beautiful colorful shapes">
+            </div>
```

Comment:
- [ ] As per the design, kindly delete this image and use it as a background instead.
Hint: You can create an empty div, and apply this image as background, to ease positioning